features/cellular/framework/targets/QUECTEL/BG96/QUECTEL_BG96_CellularContext.cpp

```diff
@@ -69,11 +69,20 @@ ControlPlane_netif *QUECTEL_BG96_CellularContext::get_cp_netif()
 
 nsapi_error_t QUECTEL_BG96_CellularContext::do_user_authentication()
 {
+    uint8_t type = 1;
+    if ((uint8_t)_pdp_type > 1) {
+        type = 2;
```

Comment:
BG96 TCP/IP AT Commands Manual version 1.0 says that IPv6 is still _under development_ for `AT+QICSGP`?

Reply:
Answering to my question above, [BG96_TCP/IP_AT_Commands_Manual_V1.1](https://www.quectel.com/ProductDownload/bg96.html) has support for IPv4v6, so type 3 shall be accepted here as well.

Reply:
Added